eth/v1alpha1/attestation.proto

```diff
@@ -34,14 +34,14 @@ message Attestation {
 }
 
 message AggregateAttestationAndProof {
-    // index is the aggregator's index
-    uint64 index = 1;
+    // The aggregator index that submitted this aggregated attestation and proof.
+    uint64 aggregator_index = 1;
 
-    // selection_proof is the signature of the slot aggregator signed to
-    bytes selection_proof = 2;
-
-    // aggregate is the constructed aggregated attestation
+    // The aggregated attestation that was submitted.
     Attestation aggregate = 3;
+
+    // The selection proof signed by the aggregator, which is the signature of the slot to aggregate.
```

Comment:
Please add the bytes size. Is this 96 bytes signature?

Reply:
will do

Reply:
https://github.com/prysmaticlabs/ethereumapis/commit/fe5092480553aea2051650c4a4209421fa7d90ec